docs/source/deployment/deploy_sre.md

```diff
@@ -69,6 +69,48 @@ sre:
 
 ::::
 
+:::{admonition} Supported Azure regions
+:class: dropdown important
+
+Some of the SRE resources are not available in all Azure regions.
+
+- Workspace virtual machines use zone redundant storage managed disks which have [limited regional availability](https://learn.microsoft.com/en-us/azure/virtual-machines/disks-redundancy).
+- Some shares mounted on workspace virtual machines require premium file shares which have [limited regional availability](https://learn.microsoft.com/en-us/azure/storage/files/redundancy-premium-file-shares).
+
+The regions which satisfy all requirements are,
```

Comment:
If we're prepared to be this definitive here, are we also prepared to enforce this in pydantic validation?